release-notes/release-notes52.md

```diff
@@ -83,6 +84,26 @@ A new `link_default_protocol` option has been added to the Autolink and Link plu
 - For information on using this setting with the Autolink plugin, see: [Autolink plugin - `link_default_protocol`]({{site.baseurl}}/plugins/autolink/#link_default_protocol).
 - For information on using this setting with the Link plugin, see: [Link plugin - `link_default_protocol`]({{site.baseurl}}/plugins/link/#link_default_protocol).
 
+## Minor changes for TinyMCE 5.2
+
+{{site.productname}} 5.2 introduces the following minor changes:
+
+- Adds the ability to apply formats to spaces.
+- Adds a drop shadow below the toolbar while in sticky mode and introduced Oxide variables to customize it when creating a custom skin.
+- Adds a `quickbars_image_toolbar` setting to allow for the image quickbar to be turned off.
+- Adds the `loading` attribute to the default schema for `iframe` and `img` . Patch contributed by [ataylor32](https://github.com/ataylor32).
+- Adds new `getNodeFilters` and `getAttributeFilters` functions to the `editor.serializer` instance.
+- Adds error message events that fire when a resource loading error occurs.
+- Changes the default schema to disallow `onchange` for select elements.
```

Comment:
This was for security reasons iirc? Would we wanna say that?

Reply:
- Improved security by changing the default schema to disallow `onchange` for select elements.